packages/components/src/range-control/stories/index.tsx

```diff
@@ -0,0 +1,243 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { starEmpty, starFilled, styles, wordpress } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import RangeControl from '..';
+
+const ICONS = { starEmpty, starFilled, styles, wordpress };
+
+const meta: ComponentMeta< typeof RangeControl > = {
+	component: RangeControl,
+	title: 'Components/RangeControl',
+	argTypes: {
+		afterIcon: {
+			control: { type: 'select' },
+			options: Object.keys( ICONS ),
+			mapping: ICONS,
+		},
+		beforeIcon: {
+			control: { type: 'select' },
+			options: Object.keys( ICONS ),
+			mapping: ICONS,
+		},
+		color: { control: { type: 'color' } },
+		help: { control: { type: 'text' } },
```

Comment:
Since dashicons are not loaded in Storybook, it's probably better to disable this control?

```suggestion
		help: { control: { type: 'text' } },
		icon: { control: { type: null } },
```

Reply:
✅ Done, thanks.